app/controllers/SurveyController.java

```diff
@@ -164,7 +201,8 @@ public Result versionSurvey(String surveyGuid, String createdOnString) throws Ex
         
         Survey survey = surveyService.getSurvey(keys);
         verifySurveyIsInStudy(session, study, survey);
-        
+
+        // This creates a new version of the survey, and so, no cache clearing needs to occur
```

Comment:
Won't this impact the most recent and published versions of the survey? (In the sense that they will now be pointing to different versions?)